keanu-project/src/test/java/io/improbable/keanu/vertices/dbltensor/nonprobabilistic/unary/UnaryOperationTestHelpers.java

```diff
@@ -0,0 +1,71 @@
+package io.improbable.keanu.vertices.dbltensor.nonprobabilistic.unary;
+
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.tensor.dbl.Nd4jDoubleTensor;
+import io.improbable.keanu.vertices.dbltensor.DoubleTensorVertex;
+import io.improbable.keanu.vertices.dbltensor.nonprobabilistic.ConstantTensorVertex;
+import io.improbable.keanu.vertices.dbltensor.nonprobabilistic.diff.TensorDualNumber;
+import io.improbable.keanu.vertices.dbltensor.probabilistic.TensorUniformVertex;
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+import static org.junit.Assert.assertEquals;
+
+public class UnaryOperationTestHelpers {
+
+    public static void operatesOnScalarVertexValue(double aValue,
+                                                   double expected,
+                                                   Function<DoubleTensorVertex, DoubleTensorVertex> op) {
+
+        TensorUniformVertex A = new TensorUniformVertex(0.0, 1.0);
```

Comment:
Can you use `... A = new ConstantTensorVertex(aValue)` here?

Reply:
Legend, can do!